app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -38,19 +38,22 @@
           android:paddingTop="@dimen/add_profile_main_padding_top"
           android:paddingBottom="@dimen/add_profile_main_padding_bottom">
 
-          <com.jackandphantom.circularimageview.CircleImage
+          <com.google.android.material.imageview.ShapeableImageView
             android:id="@+id/add_profile_activity_user_image_view"
             android:layout_width="120dp"
             android:layout_height="120dp"
             android:clickable="true"
             android:contentDescription="@string/current_profile_picture_content_description"
             android:focusable="true"
             app:add_shadow="true"
+            app:strokeColor="@color/avatar_border"
+            app:strokeWidth="1dp"
```

Comment:
These properties weren't here before--did you mean to add them?

Reply:
Apologies , I forgot to stash these changes.
I have removed them.